NickvisionMoney.Shared/Resources/Strings.zh.resx

```diff
@@ -0,0 +1,643 @@
+		<value>顏色</value>
+	</data>
+
+	<data name="DestinationAccount.Field" xml:space="preserve">
+		<value>目標戶口</value>
+	</data>
+
+	<data name="DestinationAccount.Placeholder" xml:space="preserve">
+		<value>請選擇目標戶口</value>
+	</data>
+
+	<data name="DestinationAccount.Invalid" xml:space="preserve">
+		<value>目標戶口 (無效)</value>
+	</data>
+
+	<!--Settings-->
+	<data name="UserInterface" xml:space="preserve">
+		<value>介面</value>
+	</data>
+
+	<data name="UserInterfaceDescription" xml:space="preserve">
+		<value>自訂介面</value>
+	</data>
+
+	<data name="Theme" xml:space="preserve">
+		<value>主題</value>
+	</data>
+
+	<data name="ThemeLight" xml:space="preserve">
+		<value>淺色主題</value>
+	</data>
+
+	<data name="ThemeDark" xml:space="preserve">
+		<value>暗色主題</value>
+	</data>
+
+	<data name="ThemeSystem" xml:space="preserve">
+		<value>按系統主題</value>
+	</data>
+
+	<data name="Colors" xml:space="preserve">
+		<value>顏色</value>
+	</data>
+
+	<data name="ColorsDescription" xml:space="preserve">
+		<value>自訂介面的顏色</value>
+	</data>
+
+	<data name="TransactionColor" xml:space="preserve">
+		<value>預設交易顏色</value>
+	</data>
+
+	<data name="TransferColor" xml:space="preserve">
+		<value>預設轉帳顏色</value>
+	</data>
+
+	<data name="TransactionColorDescription" xml:space="preserve">
+		<value>改變此設定僅會影響新增之交易</value>
+	</data>
+
+	<data name="TransferColorDescription" xml:space="preserve">
+		<value>改變此設定僅會影響新增之轉帳</value>
+	</data>
+
+	<!--Shortcuts Dialog-->
+	<data name="Application.Shortcut" xml:space="preserve">
+		<value>軟體</value>
+	</data>
+
+	<!--WinUI Specific-->
+	<data name="FixErrors.WinUI" xml:space="preserve">
+		<value>請修復上述錯誤</value>
+	</data>
+
+	<data name="ChangelogTitle.WinUI" xml:space="preserve">
+		<value>新增功能</value>
+	</data>
+
+	<data name="CreditsDialogDescription.WinUI" xml:space="preserve">
+		<value>開發人員:&#xA;{0}&#xA;&#xA;設計師:&#xA;{1}&#xA;&#xA;藝術家:&#xA;{2}&#xA;&#xA;翻譯人員:&#xA;{3}</value>
+	</data>
+
+	<data name="ThemeDescription.WinUI" xml:space="preserve">
+		<value>請重啟以應用設定</value>
+	</data>
+
+	<data name="RestartThemeTitle.WinUI" xml:space="preserve">
+		<value>重啟以應用新主題?</value>
+	</data>
+
+	<data name="RestartThemeDescription.WinUI" xml:space="preserve">
+		<value>確認重啟 {0} 以應用新主題?&#xA;任何未儲存的進展都將丟失</value>
+	</data>
+
+	<!--GTK Specific-->
+	<data name="AccountMenu.GTK" xml:space="preserve">
+		<value>帳戶選單</value>
+	</data>
+
+	<data name="ToggleSidebar.GTK" xml:space="preserve">
+		<value>切換側邊欄</value>
+	</data>
+
+	<data name="MainMenu.GTK" xml:space="preserve">
+		<value>主選單</value>
+	</data>
+
+	<data name="Recents.GTK" xml:space="preserve">
+		<value>最近開啟</value>
+	</data>
+
+	<data name="NewAccountPopover.GTK" xml:space="preserve">
+		<value>新增</value>
+	</data>
+
+	<data name="CloseAccount.GTK" xml:space="preserve">
+		<value>關閉帳戶</value>
+	</data>
+
+	<data name="AccountActions.GTK" xml:space="preserve">
+		<value>動作</value>
+	</data>
+
+	<data name="AccountFileFilter.GTK" xml:space="preserve">
+		<value>帳戶 (*.nmoney)</value>
```

Comment:
And here it's probably the same situation

Reply:
It's pretty confusing to say "money bank account". If we had a new name #139  these could be easily changed.

Reply:
For now leave it as is, the name change will be implemented before the stable release so we'll finalize the translation then.